### DISCUSSION TITLE
[Projects Beta] using teams in filters

### DISCUSSION THREAD

**Discussion by ssadowski-uturn** [community/community#50703] [Projects and Issues]
### Select Topic Area

General

### Body

Did not find this but my search turned up nothing, though it could have been too broad

Challenge: I have a view that should only be showing active issues for a specific team.

Current workaround: using the assignee filter, I can try and limit scope to people on the team, however that's a decently large number of people and the filter doesn't work when it's larger than 255 chars.

Preferred method: a 'team:' filter. Everyone is assigned to a team, and if I can use that it solves the problem, however the issues aren't assigned to a team - so something like ```assignee: team=teamname``` would be my preference. In reality, this should be doable for the reviewers as well so ```reviewer: team=teamname```

Current state of github would require that to be evaluated as people who are in the team, since AFAIK issues can't be assigned to teams but review requests can, this would allow reviewer to be people who are in the team or the team itself while keeping the syntax the same for both fields.

**Comment by beingminimal** [comment]
@queenofcorgis Any update on this feature request?

**Comment by opegamanalitarok** [comment]
Currently, GitHub does not have a built-in filter for issues assigned to a specific team. As you mentioned, the assignee filter can be used to limit the scope to people on the team, but it has limitations. Another workaround could be to use labels to indicate which team the issue belongs to, and then filter by that label.

However, if you want to filter by team, one possible solution is to use a third-party tool or an API. For example, you can use the GitHub API to retrieve a list of issues assigned to members of a specific team, and then filter those results. Another option is to use a third-party tool like Zapier, which allows you to create custom filters based on various criteria, including team membership.

Alternatively, you can create a custom script that uses the GitHub API to fetch issues assigned to members of a specific team and create a new view with those issues. However, this would require some programming skills.

Hope this helps! 🙌🙌
